www-client/firefox-bin/firefox-bin-73.0-r1.ebuild

```diff
@@ -134,14 +139,13 @@ src_install() {
 		MOZ_INSTALL_L10N_XPIFILE="1" \
 		mozlinguas_src_install
 
-	if use alsa ; then
-		local apulselib="/usr/$(get_libdir)/apulse"
+	if use alsa && ! use pulseaudio; then
 		patchelf --set-rpath "${apulselib}" "${ED}"${MOZILLA_FIVE_HOME}/libxul.so || die
 	fi
 
 	# Create /usr/bin/firefox-bin
 	dodir /usr/bin/
-	local apulselib=$(usex pulseaudio "/usr/$(get_libdir)/apulse:" "")
+	local apulselib=$(usex pulseaudio "" $(usex alsa "/usr/$(get_libdir)/apulse:" ""))
```

Comment:
Although after reverting the logic for `usex pulseaudio` this became unneedingly complicated, and in its current state can just be added with `usex alsa`.

Reply:
I find this hard to read.

Reply:
It was made this way because I followed the previous logic, `usex pulseaudio "/usr/$(get_libdir)/apulse:" ""` which then didn't seem to make any sense. I believe simple `$(usex pulseaudio "/usr/$(get_libdir)/apulse:" "")` was the intention and should work all right now.